### ISSUE TITLE
Invalid Ref Call Attempt

### ISSUE THREAD

**Issue by deviljin112** [brouznouf/fivem-mysql-async#41]
Hello,

After the update, for some strange reason any admin command i use (i.e /car or /tpwaypoint etc) does not work and in console i get 

`Invalid ref call attempt: 13`

The number changes based on the command used...

Any ideas?

**Comment by Bazookan** [comment]
I have this exact same issue aswell. After the 3.0.1 update!

**Comment by deviljin112** [comment]
So i found 2 fixes... (either is bad)

1. If you restart esx_status the errors stop -> breaks esx_status (in my case it didnt work anyway after update)
2. Turn off all scripts that have their client coding in .js (not in html folder or nui) -> for some reason esx_status works but acts strange and sometimes breaks

Thats all i concluded so far...

**Comment by GHMatti** [comment]
Might be fixed by 3.0.2, if this was caused by 3.0.1. There were some unsafe changes in the MySQL.lua

If it still persists it might be something else.

**Comment by deviljin112** [comment]
Sooooo also found out recently update 3.0.1 breaks multi query, so that means stuff like gcphone stops working because it requires two queries running simultaneously, async exceeds my programming knowledge and skills so I'm literally waiting and hoping the author knows how to solve all these issues or has tips on how to

**Comment by GHMatti** [comment]
Did you use 3.0.2? Which should fix the issue with the phone.

It originally did not allow multiple statements per query.

**Comment by deviljin112** [comment]
Oh no i havent i actually totally missed you've updated it 2hrs ago hahaha I will check it out tomorrow currently unable to access my computer I will update you when I check it out thanks

**Comment by GrapsZ** [comment]
3.0.2 doesn't fix the issue with the phone dude :/

**Comment by GHMatti** [comment]
Which connection string are you using?

**Comment by CruelAlpha** [comment]
This issue does nto seem to be fixed yet

**Comment by deviljin112** [comment]
Just for the sake of the argument and for those looking for a fix...

Async DOES NOT cause invalid ref attempt... That was caused by newest fivem server update that caused issues with client.js scripts... The only thing that async has broken (that is now apparently fixed - I haven't tested that myself) is multistatement dB entries which were required by scripts such as gcphone... Therefore, async caused errors with gcphone and multistatement... And fivem server artifact caused admin commands breaking and invalid statement so don't expect the author to fix what this script has not broken and allow him to continue his great work on async...

Thank you

**Comment by GHMatti** [comment]
yes. And the title is about the ref calls. so it was closed, as I am not fond to have an open issue about another open issue, that might not be open at all.

https://github.com/citizenfx/fivem/commit/b250df1cf1859e21d8a2078b23ade14e098a6ebd
